bin/check_metric_limits.rb

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env ruby-2.1.1
```

Comment:
It should be possible to run with either of those too. I just used this because the default 'ruby' on our ADS servers is ruby 1.8.7, which wasn't working for me. As this is only a helper script (it's not run automatically by anything, and most users won't need it either) I wasn't too concerned about locking it to a specific Ruby version for now.